operator/bundle_helpers/patch-csv.py

```diff
@@ -113,6 +114,7 @@ def parse_args():
 
 
 def main():
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Let's explicitly tell it should log to `stderr` so that I can sleep well at night:
```suggestion
    logging.basicConfig(stream=sys.stderr, level=logging.INFO)
```